R/validation.R

```diff
@@ -4,14 +4,14 @@ is_rectangular <- function(.data){
   is_rectangular_sample =
     .data %>%
     count(sample) %>%
-    count(`n`, name = "nn") %>%
+    count(dplyr::n, name = "nn") %>%
```

Comment:
the n here is a column name, not a function. Any particular reson you are using this form? Daos it fail on your computer otherwise?

Reply:
I put it in the commit description, with just `n` it throws an error with BiocCheck. This is one way to fix the error do you know any other?

Reply:
actually I can just add `@importFrom dplyr n` instead

Reply:
> 
> 
> I put it in the commit description, with just `n` it throws an error with BiocCheck. This is one way to fix the error do you know any other?

Strange it does not on my biocheck. Do you have Bioconductor 3.12 or 3.11 installed?

Reply:
3.12. What BiocCheck version do you have? I updated to latest 1.25.5

Reply:
This is BiocCheck version 1.25.8. and I did not get error. so strange :)

Reply:
Yes strange, I will test my end with BiocCheck version 1.25.8. Yep good to use 3.12 as that's what we're supposed to develop against.

Reply:
Ok that's interesting, no error with BiocCheck 1.25.8. But 1.25.8. is the GitHub biocheck version that's not released yet & I guess they might use the Bioc devel version to test packages? which is 1.25.5 https://bioconductor.org/packages/devel/bioc/html/BiocCheck.html
should I remove the change if we think 1.25.8 will be what they use to check and if it throws the error we say it works with 1.25.8?